deploy/60_deployment.yaml

```diff
@@ -52,6 +52,8 @@ spec:
         env:
         - name: ENABLE_STATUS_REPORTING
           value: "true"
+        - name: ENABLE_UPGRADEPOLICY_STATUS
+          value: "true"
         volumeMounts:
```

Comment:
@supreeth7  adding this here as the CSV generation for all env will be picked from here .